proxy/slither.config.json

```diff
@@ -1,4 +1,4 @@
 {
-    "detectors_to_exclude": "uninitialized-local,calls-loop,reentrancy-events,assembly,solc-version,timestamp,unused-return,variable-scope,dead-code,erc20-interface",
+    "detectors_to_exclude": "uninitialized-local,calls-loop,reentrancy-events,assembly,solc-version,timestamp,unused-return,variable-scope,dead-code,erc20-interface,uninitialized-state",
```

Comment:
I don't like disabling of the `uninitialized-state` warning. Prefer to initialize state variables.

Reply:
I will try to turn it on back, but false-positive gives incorrect result here

Reply:
Slither makes warning on `_registryContracts` mapping which is initialized in a different contract